about.html

```diff
@@ -0,0 +1,123 @@
+            experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+            This year, we hope to expand our invitation list including organizations and individuals.</p>
+        <h6 class="headline-address">Please contact us per Email for any firther questions about Tech Summit 2023</h6>
+        <h6 class="headline-email">email@domain.com</h6>
+    </section>
+    <!-- Headline Section Ends Here -->
+
+     <!-- Logo Selection Section Starts Here -->
+    <!--  -->
+    <section class="logo-selection">
+        <h3>Global Tech Summit 2023 Logo</h3>
+        <hr>
+        <p>The logo of Tech Summit 2023 was decided through logo competition from 7. July - 12. July</p>
+        <div class="logo-selection__logo">
+            <img src="./assets/Logopit_1684831391238.png" alt="">
+        </div>
+    </section>
+    <hr class="hr-full">
+    
+    <!-- Logo Selection Section Ends Here -->
+
+    <!-- Past Summits Section Starts Here -->
+    
+    <section class="past-summits">
+        <h3>See the past Tech Summits</h3>
+        <hr>
+        <p>Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+        <div class="past-summit-container">
+            <img src="./assets/pexels-matheus-bertelli-3321802.jpg" alt="Avatar" class="past-summit-bg">
+            <div class="past-summit-data">
+                <h4>2020</h4>
+                <p>Tech Summit 2020 Warsaw</p></div>
+          </div>
+          <div class="past-summit-container">
+            <img src="./assets/pexels-matheus-bertelli-3321797.jpg" alt="Avatar" class="past-summit-bg">
+            <div class="past-summit-data">
+                <h4>2020</h4>
+                <p>Tech Summit 2020 Aires</p></div>
+          </div>
+    </section>
+    
+    <!-- Past Summits Section Ends Here -->
+
+    <!-- Partners Section Starts Here -->
+    <section class="partners">
+        <h3>Partners</h3>
+        <hr>
+        <div class="partners__body">
+            <img src="./assets/partners/mozilla 1.png" alt="">
+            <img src="./assets/partners/googlekorea 1.png" alt="">
+            <img src="./assets/partners/naver 1.png" alt="">
+            <img src="./assets/partners/daumkakaologo 1.png" alt="">
+            <img src="./assets/partners/airbnb 1.png" alt="">
+        </div>
+    </section>
+    <!-- Partners Section Ends Here -->
+
+</main>
+
+     <!-- Footer Section Starts Here -->
+     <footer>
+        <div class="footer__logo">
+            <img src="./assets/Logopit_dark.png" alt="logo">
+        </div>
+        <div class="footer-body">
+            <h6 class="footer-body__heading">2023 Global Tech India, Some Rights Reserved</h6>
+            <!-- <p class="footer-body__description">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Error asperiores nemo sint ipsa suscipit animi.</p> -->
+        </div>
+    </footer>
```

Comment:
- [ ] While you make a great effort I see that the image at the footer not working ,so it will be better if you fix that you can check if the path is correct for the image to make the image of footer like the Behance design 💯🚀

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|   ![image](https://github.com/suthish777/global-summit/assets/83360636/283447fc-2d49-4028-b0b3-0ea8c207d319)  |   ![image](https://github.com/suthish777/global-summit/assets/83360636/8712fc7e-180f-4061-8396-6f46aa167e60)|